dashboards-observability/public/components/visualizations/charts/pie/pie.tsx

```diff
@@ -29,7 +35,23 @@ export const Pie = ({ visualizations, layout, config }: any) => {
     valueSeries = defaultAxes.yaxis || take(fields, lastIndex > 0 ? lastIndex : 1);
   }
 
+  const invertHex = (hex:string) => {
```

Comment:
You may remove the brackets if there's only one return statement.

Reply:
@mengweieric removed brackets.